File: tests/go/cases/case_215_closure_registry_delay_02.go
package escape_tests

var retainedCase215 = []map[string]string{}

func Case215ClosureRegistryDelay02(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "closure_registry_delay_02",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "closure_registry_delay_02:" + raw,
	}
	handler := func() string { retainedCase215 = append(retainedCase215, payload); return payload["input"] }
	_ = handler
	// ESCAPE: retained closure-like dispatch.
	retainedCase215 = append(retainedCase215, map[string]string{"h": payload["task"]})
	return "ok"
}

File: tests/nodejs/cases/case_215_closure_registry_delay_02.js
/**
 * Task case 215: closure_registry_delay_02 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case215ClosureRegistryDelay02(input) {
  const taskName = 'closure_registry_delay_02';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const later = () => { retainedAudit.push(payload); return payload.input; };
  // ESCAPE: retained closure executes after caller returns.
  retainedHandlers.push(later);
  return 'ok';
}

module.exports = { case215ClosureRegistryDelay02 };
